src/lib/utils/formatter/camelToSnake.ts

```diff
@@ -1,37 +1,48 @@
-import mapObject from "map-obj";
```

Comment:
can we remove `map-obj` from our package?

Reply:
its included as dependency in
`camelcase-keys`
`decamelize-keys`